src/site/_includes/partials/topic.njk

```diff
@@ -1,23 +1,27 @@
-<div class="w-topic">
-  <h2 id="{{ topic.title | i18n(locale) | slug }}" class="no-link w-headline--three w-topic__header">
-    {{ topic.title | i18n(locale)}}
-    <a class="w-headline-link" href="#{{ topic.title | i18n(locale) | slug }}" aria-hidden="true">#</a>
-  </h2>
-  <ul class="w-path-list">
-    {% if topic.pathItems %}
-      {% set section = topic %}
-      {% include 'partials/item.njk' %}
-    {% endif %}
-  </ul>
+<div class="wrapper pad-block-size-{{"2" if loop.first else "1"}}">
+  <div class="prose">
+    <h2 id="{{ topic.title | i18n(locale) | slug }}" class="text-size-2 headline-link">
```

Comment:
I don't think headline-link class is defined anymore

Reply:
Yup, it you're right. I know I had seen it before, I guess it was removed at some point and I missed it. Nice catch!